packages/animations/src/players/animation_group_player.ts

```diff
@@ -148,12 +148,13 @@ export class AnimationGroupPlayer implements AnimationPlayer {
   }
 
   getPosition(): number {
-    let min = 0;
-    this.players.forEach(player => {
-      const p = player.getPosition();
-      min = Math.min(p, min);
-    });
-    return min;
+    const longestPlayer =
+        this.players.reduce((longestSoFar: AnimationPlayer|undefined, player: AnimationPlayer) => {
+          const newPlayerIsLongest =
+              longestSoFar === undefined || player.totalTime > longestSoFar!.totalTime;
```

Comment:
nit: the non-null operator should be unnecessary due to the narrowing effect of the prior term.
```suggestion
              longestSoFar === undefined || player.totalTime > longestSoFar.totalTime;
```